app/Controller/ServersController.php

```diff
@@ -1266,6 +1266,88 @@ private function __checkVersion()
         }
     }
 
+    public function idTranslator() {
+
+        // The id translation feature is limited to people from the host org
+        if (!$this->_isSiteAdmin() && $this->Auth->user('org_id') != Configure::read('MISP.host_org_id')) {
+            throw new MethodNotAllowedException(__('You don\'t have the required privileges to do that.'));
+        }
+
+        //We retrieve the list of remote servers that we can query
+        $options = array();
+        $options['conditions'] = array("pull" => true);
+        $servers = $this->Server->find('all', $options);
```

Comment:
You can use 'list' instead of 'all': see https://book.cakephp.org/2/en/models/retrieving-your-data.html#find-list

Reply:
I use the same array later in the code and I need to use the URL at that point which I couldn't get via list I think

Reply:
True, my mistake.